test/basic.test.js

```diff
@@ -31,6 +31,7 @@ function fakePlayer (pos) {
     },
     jumpTicks: 0,
     jumpQueued: false,
+    entities: {},
```

Comment:
You can give a list directly to simplify the interface, you don't use the key anyway

Reply:
I made `entities` an obj to more emulate a real player, but I'll change it to an array